sdks/java/container/boot.go

```diff
@@ -20,19 +20,20 @@ package main
 import (
 	"context"
 	"flag"
-	"fmt"
 	"log"
 	"os"
 	"path/filepath"
 	"strconv"
 	"strings"
 
 	"github.com/apache/beam/sdks/go/pkg/beam/artifact"
-	pb "github.com/apache/beam/sdks/go/pkg/beam/model/fnexecution_v1"
+	fnpb "github.com/apache/beam/sdks/go/pkg/beam/model/fnexecution_v1"
+	pb "github.com/apache/beam/sdks/go/pkg/beam/model/pipeline_v1"
```

Comment:
why this selection for name, out of curiosity?

Reply:
I don't have a better one for the basic data structure package.

Reply:
I'd prefer something like `pmpb`, but if you feel strongly I'm not extremely concerned

Reply:
Ack. I prefer just pb.